### PR TITLE
Control Panel Documentation

### LINKED ISSUE #12: Control Panel Subsystem
- [x] List of Inputs (Sensors)
- [x] List of Outputs (Motors, Pneumatics)
    - [x] `RobotMap`
- [x] List of Actions (Commands)
- [x] Integrate Commands into Robot class

### PR DESCRIPTION
**Pull request by jeffaburke**
Closes #12

### CODE REVIEW

File: src/main/java/frc/robot/subsystems/Lift.java [hunk 1]
```diff
@@ -7,6 +7,11 @@
 import edu.wpi.first.wpilibj2.command.SubsystemBase;
 import frc.robot.maps.RobotMap;
 
+//Lifts the robot to be able to climb the bar in the last 30 seconds of the game
```
Comment (bot190) [inline src/main/java/frc/robot/subsystems/Lift.java]: Please remove these changes if they are in the lift PR.

File: src/main/java/frc/robot/subsystems/ControlPanel.java [hunk 2]
```diff
@@ -7,28 +7,65 @@
...
     }
 
+    /**
+     * Spin Forwards
+     * 
+     * This command runs the motor in a positive set direction
+     * 
+     * @return {@link StartEndCommand} for running the motor at spinnerMotorSpeed
+     */
     public CommandBase spinForwards() {
```
Comment (bot190) [inline src/main/java/frc/robot/subsystems/ControlPanel.java]: This command and `spinBackwards` should be replaced with commands to spin the control panel a specific number of times and to a specific color.

**Review by bot190 — CHANGES_REQUESTED** [review]
I don't think the generic movement commands are very useful for this subsystem.

File: src/main/java/frc/robot/subsystems/ControlPanel.java [hunk 3]
```diff
@@ -3,34 +3,66 @@
...
 public class ControlPanel extends SubsystemBase {
 
-    private SendableSpeedController spinnerMotor;
+    private final SendableSpeedController spinnerMotor;
 
     private static final double spinnerMotorSpeed = 1;
 
-    public ControlPanel(RobotMap.ControlPanelMap map) {
+    /**
+     * Defines a motor
```
Comment (msoucy) [inline src/main/java/frc/robot/subsystems/ControlPanel.java]: Does not define a motor - pulls it from the robot map

**Review by msoucy — CHANGES_REQUESTED** [review]

**Pull request closed without merging**